server/src/transactions/dto/create-transaction.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { IsNotEmpty, IsEnum } from 'class-validator';
+import { TransactionStatus } from '../transaction-status.enum';
+import { User } from 'src/users/entities/user.entity';
+import { Organization } from 'src/organizations/entities/organization.entity';
+import { Asset } from 'src/assets/entities/asset.entity';
+
+export class CreateTransactionDto {
+  @IsNotEmpty()
+  donater_user: User;
+
+  donater_organization: Organization;
```

Comment:
optional here i think no?

Reply:
I was talking to Kevin about this, and we thought that since a Transaction had to have a donater, it should probably be a donater_user who could optionally also have a donater_organization id.   If both donater_user and donater_organization were optional we could have a transaction with no donater.   Does that make sense?

Reply:
yep, i was referring to just the `donater_organization`. sorry for not being clear